tests/cpp/test_utils.cpp

```diff
@@ -2072,4 +2074,84 @@ TEST_F(TestCpp23BackPort, Enumerate) {
   // bidirectional
 }
 
+namespace {
+
+// Generator that yields integers from 0 to n-1
+Generator<int> count(int n) {
```

Comment:
Regarding naming, "count" is overloaded. Consider something like `zeroToN` or `integerRange`.

Reply:
changed